docs/src/main/sphinx/functions.rst

```diff
@@ -8,7 +8,9 @@ Trino operating on the underlying data sources.
 
 If you are looking for a specific function or operator, see the :doc:`full
 alphabetical list</functions/list>` or the :doc:`full list by
-topic</functions/list-by-topic>`.
+topic</functions/list-by-topic>`. Using :doc:`SHOW FUNCTION
+</sql/show-functions>` may yield information on additional arguments and
```

Comment:
dont hedge with "may yield" ... also just be specific on what it returns ... a list of all functions with the supported arguments and a short description .. also maybe worth pointing out that this is in an SQL console connected to Trino .. so its kind of built-in help